Dapper.Tests.Performance/Massive/Massive.cs

```diff
@@ -323,7 +323,6 @@ public virtual DbCommand CreateUpdateCommand(object o, object key)
             var settings = (IDictionary<string, object>)expando;
             var sbKeys = new StringBuilder();
             const string stub = "UPDATE {0} SET {1} WHERE {2} = @{3}";

```

Comment:
```suggestion
            const string stub = "UPDATE {0} SET {1} WHERE {2} = @{3}";
            var args = new List<object>();
```
This is modifying Massive - a source package. It shouldn't be touched.